backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java

```diff
@@ -724,4 +724,16 @@ public static void validateEntityNameWithColonAndSlash(String name) throws Model
           "Name can not contain ':' or '/' or '\\\\'", Code.INVALID_ARGUMENT);
     }
   }
+
+  public static ModelDBException getInvalidFieldException(IllegalArgumentException ex) {
+    if (ex.getCause().getMessage().contains("could not resolve property: ")) {
```

Comment:
lets add a null check on ex.getCause() and ex.getCause().getMessage()

Reply:
Added null check on ex.getCause(). No need to add null check on ex.getCause().getMessage() because if ex.getCause() is available then always some message is there.